src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -86,7 +86,15 @@
     static {
         Properties prop = new Properties();
         try {
-            prop.load(new FileInputStream("src/test/resources/test.properties"));
+            String propertiesFilename;
+            if (isTravis()) {
+                propertiesFilename = "test.travis.properties";
+            } else if (isAppveyor()) {
+                propertiesFilename = "test.appveyor.properties";
+            } else {
+                propertiesFilename = "test.properties";
+            }
+            prop.load(new FileInputStream("src/test/resources/" + propertiesFilename));
```

Comment:
If we hardcode the AppVeyor chromedriver binary path into the template, it feels misleading. It's also not very explicit that it's the AppVeyor path, as it's found in the general template.

I quite strongly feel that a template file should be used as a reference and not be depended upon to provide essential values in a build environment. What if we decide to switch Travis to Chrome too in future? We'd need two paths to the chromedriver binary that are specific to their build environments.

Besides, may I ask what inhibitions you have about creating separate properties files per build environment? I find this very clear and readable as we'd know exactly what test settings we have for each environment, without delving into the code of `TestProperties`.